--- migrations.rs.orig	2023-01-01 00:00:00.000000000 +0000
+++ migrations.rs	2023-01-01 00:00:00.000000000 +0000
@@ -65,6 +65,11 @@
     impl<T: Config> pallet_session::migrations::v1::MigrateDisabledValidators
         for MigrateDisabledToSession<T>
     {
+        #[cfg(not(feature = "try-runtime"))]
+        fn peek_disabled() -> sp_runtime::Vec<(u32, sp_staking::offence::OffenceSeverity)> {
+            sp_runtime::Vec::new()
+        }
+
         #[cfg(feature = "try-runtime")]
         fn peek_disabled() -> sp_runtime::Vec<(u32, sp_staking::offence::OffenceSeverity)> {
             DisabledValidators::<T>::get().into_inner()
